src/pyatsimagebuilder/utils.py

```diff
@@ -242,70 +258,85 @@ def to_image_path(path, context, workspace_dir):
     return path
 
 
-def discover_jobs(jobfiles, context, install_dir, workspace_dir):
+def discover_jobs(jobfiles, 
+                  context_path, 
+                  install_dir, 
+                  workspace_dir, 
+                  relative_path=True):
```

Comment:
`workspace_dir` is only used if `relative_path` is `True`. This could be simplified using only a single `relative_path` arg, there's no real reason to have two different args for this.
```suggestion
                  relative_path=None):
```